incubator/mission-control/charts/elasticsearch/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Please update to the latest `_helpers.tpl`.

Reply:
Done